test/unit/test_client.py

```diff
@@ -233,18 +233,18 @@ def test_load_config_for_connection_with_filename(self):
         conf = get_fixture('eapi.conf')
         pyeapi.client.load_config(filename=conf)
         cfg = pyeapi.client.config.get_connection('test1')
-        self.assertEqual(cfg['host'], '192.168.1.16')
-        self.assertEqual(cfg['username'], 'eapi')
-        self.assertEqual(cfg['password'], 'password')
+        self.assertEqual(cfg['host'], '10.16.129.149')
+        self.assertEqual(cfg['username'], 'arista')
+        self.assertEqual(cfg['password'], 'arista')
```

Comment:
Do we need these fields updated? I assume 10.16.129.149 is your dev environment.